src/Altinn.AccessManagement/Configuration/CustomTelemetryInitializer.cs

```diff
@@ -15,7 +15,7 @@ public void Initialize(ITelemetry telemetry)
         {
             if (string.IsNullOrEmpty(telemetry.Context.Cloud.RoleName))
             {
-                telemetry.Context.Cloud.RoleName = "authorization-admin";
+                telemetry.Context.Cloud.RoleName = "accessmanagement";
```

Comment:
can probably make it 'access-management' (since every other role is specified with hyphens)

Reply:
Fixed